scripts/ethereum/Dockerfile

```diff
@@ -5,8 +5,14 @@ LABEL maintainer="dev@ambire.com"
 
 WORKDIR /scripts
 
+# The Chain Id that will be passed to `ganache-cli`
+ENV CHAIN_ID=
+# The snapshot contracts directory that will be used
+ARG SNAPSHOT_CONTRACTS=
```

Comment:
Env. variables cannot be used in `COPY`, this is why we are passing a build argument of the snapshot directory that will be loaded by Ganache-cli